pages/installation/os/sles.rst

```diff
@@ -0,0 +1,122 @@
+
+.. warning:: This setup should not be done on publicly exposed servers. This guide **does not cover** security settings!
+
+
+Prerequisites
+-------------
+
+The following Patterns are use as a minimal server setup::
+
+  - Base System
+  - Minimal System (Appliances)
+  - Yast configuration packages
+
+.. warning:: This Guide makes the assumption that the firewall is disabled and communication is possible to the outside world.
+
+Taking a minimal server setup as base will need this additional packages::
+
+  $ sudo zypper install java-1_8_0-openjdk
+
+
+MongoDB
+-------
+
+Installing MongoDB on SLES should follow `the tutorial for SLES <https://docs.mongodb.com/v3.4/tutorial/install-mongodb-on-suse/>`_ from the MongoDB documentation. Add the GPG Key and the Repository before you install MongoDB::
+
+  $ sudo rpm --import https://www.mongodb.org/static/pgp/server-3.4.asc
+  $ sudo zypper addrepo --gpgcheck "https://repo.mongodb.org/zypper/suse/12/mongodb-org/3.4/x86_64/" mongodb
+  $ sudo zypper -n install mongodb-org
+
+
+
+Additionally, run these last steps to start MongoDB during the operating system's boot and start it right away::
+
+  $ sudo chkconfig mongod on
+  $ sudo systemctl daemon-reload
+  $ sudo systemctl restart mongod.service
+
+
+Elasticsearch
+-------------
+
+Graylog 2.3.x can be used with Elasticsearch 5.x, please follow the installation instructions from `the Elasticsearch installation guide <https://www.elastic.co/guide/en/elasticsearch/reference/5.6/rpm.html>`_.
+
+First install the Elastic GPG key with ``rpm --import https://artifacts.elastic.co/GPG-KEY-elasticsearch`` then add the repository file ``/etc/zypp/repos.d/elasticsearch.repo`` with the following contents::
+
+    [elasticsearch-5.x]
+    name=Elasticsearch repository for 5.x packages
+    baseurl=https://artifacts.elastic.co/packages/5.x/yum
+    gpgcheck=1
+    gpgkey=https://artifacts.elastic.co/GPG-KEY-elasticsearch
+    enabled=1
+    autorefresh=1
+    type=rpm-md
+
+followed by the installation of the latest release with ``sudo zypper install elasticsearch``.
+
+Make sure to modify the `Elasticsearch configuration file <https://www.elastic.co/guide/en/elasticsearch/reference/5.6/settings.html#settings>`__  (``/etc/elasticsearch/elasticsearch.yml``) and set the cluster name to ``graylog`` additionally you need to uncomment (remove the # as first character) the line::
+
+    cluster.name: graylog
+
+After you have modified the configuration, you can start Elasticsearch::
+
+    $ sudo chkconfig elasticsearch on
+    $ sudo systemctl daemon-reload
+    $ sudo systemctl restart elasticsearch.service
+
+
+Graylog
+-------
+
+First install the Graylog GPG Key with ``rpm --import https://git.io/vdPub`` then add the repository file ``/etc/zypp/repos.d/graylog.repo`` with the following content::
```

Comment:
Key &rarr; key

Reply:
Please don't use "obscure" URL shorteners for such an important URL as the GPG public key.

Reply:
using the Github URL for that would destroy any layout thats why i choose the official github shortener.

Reply:
your right!